exporter/redis.go

```diff
@@ -182,6 +182,11 @@ func (e *Exporter) initGauges() {
 		Name:      "command_call_duration_seconds_sum",
 		Help:      "Total amount of time in seconds spent per command",
 	}, []string{"addr", "alias", "cmd"})
+	e.metrics["slowlog_len"] = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: e.namespace,
+		Name:      "slowlog_len_sum",
```

Comment:
`slowlog_len_sum` --> `slowlog_length`

Reply:
and slowlog_count?Emmm.....I don't know which is the best....

Reply:
Let's go with `slowlog_length` for now.

Reply:
OKay~

Reply:
All fixed~ Thx your help~